set.mm

```diff
@@ -813400,6 +814075,318 @@ The modulo (remainder) operation - extension
     ( cz wcel c2 cmo co cc0 cfzo cpr 2nn zmodfzo ancoms mpan fzo0to2pr eleqtrdi
     c1 cn ) ABCZADEFZGDHFZGPIDQCZRSTCZJRUAUBADKLMNO $.
 
+  ${
+    $d A x $.  $d N x $.
+    $( If an integer is 0 modulo a positive integer, this integer must be the
+       product of another integer and the modulus.  (Contributed by AV,
+       7-Jun-2020.) $)
```

Comment:
Suggestion:

If an integer is congruent to 0 modulo a positive integer, then it must be divisible by that modulus.